### DISCUSSION TITLE
Xiaomi Mi Router HD  uboot 中断不了啊

### DISCUSSION THREAD

**Discussion by zhiqiang89** [immortalwrt/immortalwrt#1039] [Q&A]
U-Boot 2012.07 [ ,unknown] (Nov 02 2017 - 17:14:39)

smem ram ptable found: ver: 0 len: 5
DRAM:  494 MiB
d for 0:HLOS
cdp: get part failed for rootfs
Net:   MAC1 addr:78:11:dc:6:33:98
athrs17_reg_init: complete
:6:33:99
eth0, eth1
Hit any key to stop autoboot:  0 

NAND read: device 0 offset 0x800000, size 0x400000
 4194304 bytes read: OK
   Image Name:   ARM ImmortalWrt Linux-5.15.120
   Image Type:   ARM Linux Kernel Image (uncompressed)
   Data Size:    3246668 Bytes = 3.1 MiB
   Load Address: 42208000
   Entry Point:  42208000
   Verifying Checksum ... OK
OK

device nand0 <nand>, # parts = 9
 #: name                size            offset          mask_flags
 0: Bdata               0x00080000      0x00600000      0
 1: crash               0x00080000      0x00680000      0
 2: crash_syslog        0x00080000      0x00700000      0
 3: rsvd                0x000x02800000  0x03800000      0
 8: overlay             0x0a000000      0x06000000      0

active partition: nand0,0 - (Bdata) 0x00080000 @ 0x00600000

**Comment by remittor** [comment]
> Hit any key to stop autoboot: 0

If the bootloader does not respond to button presses, then the value of the `boot_wait` parameter has somehow been reset to the value `off` (or paramert `bootdelay=0`).
